dask/dataframe/tests/test_dataframe.py

```diff
@@ -44,6 +44,27 @@ def test_dataframe_doc():
     assert disclaimer in doc
 
 
+def test_dataframe_doc_from_non_pandas():
+    class Foo:
+        def foo(self):
+            """This is a new docstring that I just made up
+
+            Parameters:
+            ----------
+            None
+            """
+
+    d._bind_operator_method("foo", Foo.foo, original=Foo)
+    try:
+        doc = d.foo.__doc__
+        disclaimer = "Some inconsistencies with the Dask version may exist."
+        assert disclaimer in doc
+        assert "new docstring that I just made up" in doc
+    finally:
+        # make sure to clean up this alteration of the dd.DataFrame class
+        del dd.DataFrame.foo
```

Comment:
I appreciate the care taken here

Reply:
To be fair I only thought of doing this because there is a test later that asserts that foo is not an attr on dd.DataFrame :)

Reply:
Ha!  I guess we win for consistency then :)